docs/ref/checks.txt

```diff
@@ -431,7 +431,7 @@ The following checks are run if you use the :option:`check --deploy` option:
   a load balancer or reverse-proxy server  to redirect all connections to HTTPS.
 * **security.W009**: Your :setting:`SECRET_KEY` has less than 50 characters or
   less than 50 unique characters, or it's a Django-generated secret key
-  starting with django-insecure. Please generate a long and random
+  starting with Django-insecure. Please generate a long and random
```

Comment:
Hi there. I'm uncertain why this would be uppercased. Wouldn't it be better to use back ticks e.g. ``django-insecure``?

Reply:
Hello!
Because there is no "django-generated" in docs/spelling_wordlist and I got a spelling error. I have no idea why uppercase fixed the error. I can just add new value to spelling_wordlist.

Reply:
Ah, but I was referring to "Django-insecure", not "Django-generated." Nick's suggestion below adopts backticks as well as single-quotes for readability, which I think makes sense.